pkg/smb/nodeserver_test.go

```diff
@@ -198,41 +201,58 @@ func TestNodeExpandVolume(t *testing.T) {
 }
 
 func TestNodePublishVolume(t *testing.T) {
-	skipIfTestingOnWindows(t)
 	volumeCap := csi.VolumeCapability_AccessMode{Mode: csi.VolumeCapability_AccessMode_MULTI_NODE_MULTI_WRITER}
-	errorMountSource := "./error_mount_source"
-	alreadyMountedTarget := "./false_is_likely_exist_target"
-	smbFile := "./smb.go"
-	sourceTest := "./source_test"
-	targetTest := "./target_test"
+
+	// For CSI-Proxy(windows), the path has to be absolute and it should be within the context of value passed for
+	// csi-proxy argument: --kubelet-pod-path. In our tests we are using `C:\var\lib\kubelet\`
+	getTestPath := func(path string) string {
```

Comment:
pls move this function outside(standalone), other test cases would also needs this function, e.g. `TestNodeUnpublishVolume`